.github/workflows/nightly-longer-functional.yml

```diff
@@ -2,7 +2,7 @@ name: Nightly-Longer-Functional-Test
 
 on:
   schedule:
-    - cron: '0 5 * * *'  # Every day at midnight EST (UTC-5)
+    - cron: '0 0 * * *'  # TODO: Every day at midnight EST (UTC-5)
```

Comment:
Why are you changing this?

Reply:
Just to know that we have to reset it to nightly after we are done monitoring this fix.